### PR TITLE
Updated Packages

### LINKED ISSUE #1937: Fix `@parcel/watcher` Error AND Upgrade graphql-codegen to the latest version
**Describe the bug**

When the API is started, this error appears

```
[1] [SUCCESS] Generate to ./src/types/generatedGraphQLTypes.ts
[1] [SUCCESS] Generate outputs
[1]   Failed to import @parcel/watcher due to the following error (to use watch mode, install https://www.npmjs.com/package/@parcel/watcher):
[1] Error: Cannot find module '@parcel/watcher'
[1] Require stack:
[1] - /talawa-api/node_modules/@graphql-codegen/cli/cjs/utils/watcher.js
[1] - /talawa-api/node_modules/@graphql-codegen/cli/cjs/generate-and-save.js
[1] - /talawa-api/node_modules/@graphql-codegen/cli/cjs/cli.js
[1] - /talawa-api/node_modules/@graphql-codegen/cli/cjs/bin.js
[1] graphql-codegen --watch exited with code 0
```
**To Reproduce**

1. Download the latest version of the API
1. Install the latest API packages
2. Configure the API
3. Start the API
4. Error appears

**Expected behavior**

1. No error
2. All tests must be valid and pass
3. No other API functionality must be affected.

**Actual behavior**

- Error

**Screenshots**

- N/A

**Additional details**

- N/A

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by varshith257**
**What kind of change does this PR introduce?**

**Issue Number:**

Fixes #1937 

**Did you add tests for your changes?**

N/A

**Summary**

Added packages parcel-watcher and updated graphql-codegen packages

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Review by varshith257 — COMMENTED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "lint-staged": "lint-staged",
     "format:fix": "prettier --write \"**/*.{ts,tsx,json,scss,css}\"",
     "format:check": "prettier --check \"**/*.{ts,tsx,json,scss,css}\"",
-    "prepare": "husky",
+    "prepare": "husky init",
```
Comment (palisadoes) [inline package.json:20]: Why change this? Husky wasn't updated in the package list.

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comment

File: package.json (hunk 1 above)
Comment (varshith257) [reply package.json:20]: No, it is a merge of my PR 

- #2018

Comment (varshith257) [reply package.json:20]: This PR changes has been removed through this PR merge 

- #2046

Comment (varshith257) [reply package.json:20]: Even this PR made a lot of breaking changes.
```
  const query = {
    userId: currentUser._id.toString(),
    organizationId: organization._id.toString(),
```

It is the reason userid isn't querying and made a issue fo it

**Review by palisadoes — APPROVED** [review]

**Pull request merged**